homeassistant/components/risco/config_flow.py

```diff
@@ -93,6 +95,11 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    def __init__(self) -> None:
+        """Init the config flow."""
+        self._reauth: bool = False
```

Comment:
Maybe it's enough to check for `self._existing_entry`?

Reply:
I was debating with myself about this. I think the added bool makes this more explicit and easier to read, but I'm fine either way.

Reply:
We could rename `_existing_entry` to `_reauth_entry`?